[BUG] Webhook missing positional argiment

Seeing the following error after upgrade to 1.1.0RC0

pwnagotchi-launcher[436]: [2019-10-27 11:38:12,184] [ERROR] on_webhook() missing 1 required positional argument: 'path'

Have rebooted but issue persists.

Running on RPi0 with WaveShare V2 screen and no other attachments
Plugins installed are:

auto-update
memtemp
bt-tether
net-pos
aircrackonly
wpa-sec
onlinehashcrack
webgpsmap

--- a/pwnagotchi/ui/web.py
+++ b/pwnagotchi/ui/web.py
@@ -177,7 +177,7 @@ class Handler(BaseHTTPRequestHandler):
                 groups = matches.groups()
                 plugin_name = groups[0]
                 right_path = groups[1] if len(groups) == 2 else None
-                plugins.one(plugin_name, 'webhook', right_path)
+                plugins.one(plugin_name, 'webhook', self, right_path)
 
         else:
             self.send_response(404)

